src/pages/Page.jsx

```diff
@@ -0,0 +1,63 @@
+import { useState, useEffect } from 'react';
+
+import jsonData from '../data.json';
+import Question from '../components/Question';
+import Button from '../components/Button';
+import Loader from '../components/Loader';
+
+const Page = () => {
+  const [question, setQuestion] = useState([]);
+  const [step, setStep] = useState(0);
+
+  useEffect(() => {
+    setQuestion(jsonData);
+  }, []);
+
+  const handleNext = () => {
+    setStep(step + 1);
+  };
+
+  const handlePrev = () => {
+    setStep(step - 1);
+  };
+
+  return (
+    <div>
+      {question ? (
+        <div className="flex flex-col justify-center items-center w-full m-auto p-14 bg-gray-400 h-screen">
+          {question[step] !== undefined && (
+            <div className="py-7">
+              <Question key={question[step].id} {...question[step]} />
```

Comment:
here is props that your Question component expects 

 id: PropTypes.number.isRequired,
  question: PropTypes.string.isRequired,
  answers: PropTypes.array.isRequired,
  rightAnswer: PropTypes.number,
  
  and here what it receives {...question[step]}

Please pass expected props

Reply:
fixed